bundle/manifests/logging.openshift.io_kibanas_crd.yaml

```diff
@@ -0,0 +1,155 @@
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: kibanas.logging.openshift.io
+spec:
+  group: logging.openshift.io
+  names:
+    kind: Kibana
+    listKind: KibanaList
+    plural: kibanas
+    singular: kibana
+  scope: Namespaced
+  subresources:
+    status: {}
+  validation:
+    openAPIV3Schema:
+      description: Kibana is the Schema for the kibanas API
+      properties:
+        apiVersion:
+          description: 'APIVersion defines the versioned schema of this representation
+            of an object. Servers should convert recognized schemas to the latest
+            internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources'
+          type: string
+        kind:
+          description: 'Kind is a string value representing the REST resource this
+            object represents. Servers may infer this from the endpoint the client
+            submits requests to. Cannot be updated. In CamelCase. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#types-kinds'
+          type: string
+        metadata:
+          type: object
+        spec:
+          description: Specification of the desired behavior of the Kibana
+          properties:
+            managementState:
+              description: Indicator if the resource is 'Managed' or 'Unmanaged' by
+                the operator
+              enum:
+              - Managed
+              - Unmanaged
+              type: string
+            nodeSelector:
+              additionalProperties:
+                type: string
+              description: Define which Nodes the Pods are scheduled on.
+              type: object
+            proxy:
```

Comment:
originally called "proxySpec"

Reply:
Yes, but it was indented on the wrong place and never took its part in validation.

Reply:
it wasn't validated, but that was the field name we used. couldn't this cause regression issues?

Reply:
No there shouldn't be an issue, because validations are based on the struct tags and the field `proxySpec` used `proxy` as json struct tag.